test/test_common/environment.cc

```diff
@@ -49,6 +48,30 @@ void TestEnvironment::initializeOptions(int argc, char** argv) {
   argv_ = argv;
 }
 
+bool TestEnvironment::isTestIpVersionOnly(const Network::Address::IpVersion& type) {
+  const char* option = ::getenv("IP_TEST_TYPE");
```

Comment:
I'm thinking that setting ENVOY_IP_TEST_VERSIONS can be optional. In that case, ::getenv can return nullptr.